charts/microgateway/README.md.gotmpl

```diff
@@ -226,431 +231,372 @@ In case that the [Simple DSL configuration](#simple-dsl-configuration) does not
+In case that those objects were created with this Helm chart, just follow along with the description and configuration examples. 
+If there is already an existing Ingress or Route object and the traffic should only be passed to the Microgateway service, the information in the subchapters should provide useful information to integrate in the existing environment.
+
+:information_source: **Kubernetes vs. Openshift**:<br>
+This Helm chart can be used for Kubernetes and Openshift. While Kubernetes has "Ingress" and Openshift has "Route", simply enable the feature which fits to the environment (e.g. in Kubernetes `ingress.enabled=true` and in Openshift `route.enabled=true`).
 
 ### Kubernetes Ingress
-In the Kubernetes environments an Ingress Controller is required to make the Microgateway accessible from the Internet.  
-In our examples we will use the nginx-ingress-controller. However, this is not installed directly with the Helm chart.   
-The Helm chart only offers the possibility to create the required configuration for an existing Ingress Controller.   
-If no ingress controller is available in an environment, it can be installed with Helm.   
-
-[nginx-ingress](https://github.com/helm/charts/tree/master/stable/nginx-ingress)
-```
-helm repo add stable https://kubernetes-charts.storage.googleapis.com
-helm install nginx stable/nginx-ingress
-```
-
-#### Without TLS
-The following configuration example shows how the Ingress can be configured without TLS.  
-
-```
-ingress:
-  enabled: true
-  annotations:
-    nginx.ingress.kubernetes.io/rewrite-target: /
-    kubernetes.io/ingress.class: nginx
-  hosts:
-      - virtinc.com
-```
-
-#### TLS Configuration
-An Ingress can be protected with a Secret, which contains the TLS certificates.  
-Currently the Ingress only supports a single TLS port, 443, and assumes TLS termination.  
-If the TLS configuration section in an Ingress specifies different hosts, they are bundled on the same port according to the SNI-TLS extension.  
-
-The following configuration example shows how the Ingress can be configured with TLS.
-```
-ingress:
-  enabled: true
-  annotations:
-    nginx.ingress.kubernetes.io/rewrite-target: /
-    kubernetes.io/ingress.class: nginx
-  targetPort: https
-  tls:
-    - secretName: virtinc-tls-secret
-      hosts:
+Kubernetes allows to use different kind of Ingress controllers. Our examples are based on the [nginx-ingress](https://github.com/helm/charts/tree/master/stable/nginx-ingress) controller.
+
+  The example below shows how to install the nginx-ingress-controller with Helm:
+  ```
+  helm repo add stable https://kubernetes-charts.storage.googleapis.com
+  helm install nginx stable/nginx-ingress
+  ```
+
+:information_source: **Note**:<br>
+The Microgateway Helm chart itself does not install the nginx-ingress-controller, but allows to create an Ingress object.
+
+#### Ingress terminating HTTP
```

Comment:
I would remove terminating from the http title.

Reply:
But Ingress HTTP sounds also weird. Better suggestion?